index.js

```diff
@@ -1,95 +1,136 @@
+// => "Ahh that hits the spot"
+// => 11
+console.log("%c----------", "color: red") 
 
 
 
 // ***** Question 4 *****
 
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 4", "color: red") 
+console.log("%cQuestion 4", "color: red") 
+
+function sameSameButDifferent(num, maybeNum) {
+	if (num === maybeNum) {
+		console.log("same same")
+	}
+	else if (num == maybeNum) {
+		console.log("same same (but different)")
+	}
+	else {
+		console.log("different")
+	}
+}
 
-// console.log(sameSameButDifferent(5, 5)) 
-// // => "same same"
-//
-// console.log(sameSameButDifferent(123, "123")) 
-// // => "same same (but different)"
-//
-// console.log(sameSameButDifferent(5, 7)) 
-// // => "different"
-//
-// console.log(sameSameButDifferent(123, "122")) 
-// // => "different"
-// console.log("%c----------", "color: red") 
+console.log(sameSameButDifferent(5, 5)) 
+// => "same same"
+
+console.log(sameSameButDifferent(123, "123")) 
+// => "same same (but different)"
+
+console.log(sameSameButDifferent(5, 7)) 
+// => "different"
+
+console.log(sameSameButDifferent(123, "122")) 
+// => "different"
+console.log("%c----------", "color: red") 
 
 
 
 // ***** Question 5 *****
 
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 5", "color: red")
+console.log("%cQuestion 5", "color: red")
+
+function updateGrade(student, grade) {
+	student.grade = grade
+}
 
-// const student1 = { name: "Duane", grade: 88 }
-// updateGrade(student1, 92)
-// console.log(student1)  
-// // => { name: "Duane", grade: 92 }
-// console.log("%c----------", "color: red") 
+const student1 = { name: "Duane", grade: 88 }
+updateGrade(student1, 92)
+console.log(student1)  
+// => { name: "Duane", grade: 92 }
+console.log("%c----------", "color: red") 
 
 
 
 // ***** Question 6 *****
 
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 6", "color: red")
-
-// const users = [ 
-//   { 
-//     name: "Duane", phones: { cell: "555-123-4567", office: "555-456-7890" }
-//   },
-//   { 
-//     name: "Liza", phones: { cell: "555-234-5678", office: "555-567-1234" }
-//   }
-// ]
-// printNameAndPhones(users)
-// // => "Duane"
-// // => "Cell: 555-123-4567"
-// // => "Office: 555-456-7890"
-// // => "Liza"
-// // => "Cell: 555-234-5678"
-// // => "Office: 555-567-1234"
-// console.log("%c----------", "color: red") 
+console.log("%cQuestion 6", "color: red")
+
+const users = [ 
+  { 
+    name: "Duane", phones: { cell: "555-123-4567", office: "555-456-7890" }
+  },
+  { 
+    name: "Liza", phones: { cell: "555-234-5678", office: "555-567-1234" }
+  }
+]
+
+users.forEach(function printNameAndPhones(users) { 
+	console.log(users.name)
+	console.log(`Cell: ${users.phones.cell}`)
+	console.log(`Office: ${users.phones.cell}`)
+})
+
+// OR
+
+function printNameAndPhones(user) { 
+	console.log(user.name)
+	console.log(`Cell: ${user.phones.cell}`)
+	console.log(`Office: ${user.phones.cell}`)
+}
+
+users.forEach(printNameAndPhones)
```

Comment:
Nice demonstration of the callback syntax here! In the first example, make sure to choose good parameter names for the callback (it should probably be `user` instead of `users`, to make it clear that the function is being passed a single user). Also, you're using the `user.phones.cell` twice, instead of `user.phones.office`.

```js
users.forEach(function printNameAndPhones(user) { 
	console.log(user.name)
	console.log(`Cell: ${user.phones.cell}`)
	console.log(`Office: ${user.phones.office}`)
})
```